.circleci/config.yml

```diff
@@ -9,18 +9,29 @@ test_build_filters: &test_build_filters
     only: /.*/
   tags:
     ignore: /v[0-9]+\.[0-9]+\.[0-9]+/
-jobs:
-  test:
+executors:
+  releaser:
+    docker:
+      - image: circleci/golang:1.12
+  publisher:
     docker:
       - image: circleci/ruby:2.6.5
+  tester:
+    docker:
+      - image: circleci/ruby:2.3
+jobs:
+  test: &test-template
+    executor: publisher
     steps:
       - checkout
       - run: bundle install
       - run: rubocop -c .rubocop.yml
       - run: rake test
+  test2:
```

Comment:
Could this be named something other than `test2`? Either include the name of the ruby version (here and in `test`) or (better imho) make the test step a Matrix job which would make it easier/cleaner to add more ruby versions in the future.

Reply:
@matt-march, renamed them according to the `ruby` versions - and in order to create matrix, we must have 2-dimensional testing but here it's just 1-dimensional across only `Ruby` versions.

If you take a look at [this](https://circleci.com/blog/circleci-matrix-jobs/), you will see in the example, it's across `OS` versions x `Node` versions but here our case is different + I think, the current config is straightforward to understand how to add more `ruby` version tests in the future.